packages/semantic-ui/src/CheckboxWidget/CheckboxWidget.js

```diff
@@ -30,11 +28,7 @@ function CheckboxWidget(props) {
       inverted: false,
     },
    });
-   const displayLabel = getDisplayLabel(
-    schema,
-    uiSchema
-    /* TODO: , rootSchema */
-  );
```

Comment:
Does this not work in semantic UI?

Reply:
not suppose to use this for checkboxes. looking at the other themes,